sdcm/tester.py

```diff
@@ -249,13 +160,13 @@ def __init__(self, methodName='test', name=None, params=None,
         cluster.Setup.reuse_cluster(self.params.get('reuse_cluster', default=False))
 
         # for saving test details in DB
-        self.test_index = self.__class__.__name__.lower()
-        self.test_type = self.params.id.name
-        self.test_id = None
-        self.stats = {k: {} for k in ['test_details', 'setup_details', 'versions', 'results', 'nemesis',
-                                      'errors', 'coredumps']}
+        self._db_stats = db_stats.TestStats(self)
```

Comment:
TestStats is a perfect match for MixIn design pattern[1][2]
```
class ClusterTester(Test, TestStatsMixIn):
....
```
[1] https://en.wikipedia.org/wiki/Mixin
[2] https://www.ianlewis.org/en/mixins-and-python